aidants_connect_web/models.py

```diff
@@ -48,12 +49,28 @@ class Usager(models.Model):
 
     creation_date = models.DateTimeField(default=timezone.now)
 
+    preferred_contact_method = models.CharField(max_length=8, blank=True, null=True)
+    contact_address = JSONField(blank=True, null=True)
+    contact_email = models.EmailField(blank=True, null=True)
+    contact_phone = models.CharField(max_length=12, blank=True, null=True)
+
     def __str__(self):
         return f"{self.given_name} {self.family_name}"
 
     def get_full_name(self):
         return f"{self.given_name} {self.family_name}"
 
+    def get_contact_method(self):
+        return dict(settings.CONTACT_METHOD)[self.preferred_contact_method]
+
```

Comment:
Quand je créé un mandat
Quand je choisis de ne pas ajouter de "prefered_contact_method"
Quand je valide mon mandat
Si je vais consulter les mandats, j'ai une Key error.

Je vois trois façon de gérer cela (il y en a peut être plein d'autres) : 
- soit mettre un default à `""` dans le model pour ce champ, 
- soit de faire quelque chose comme : 
```suggestion
def get_contact_method(self):
    if self.preferred_contact_method:
        return dict(settings.CONTACT_METHOD)
    else:
        return ""
```
- Soit de mettre autre chose que None dans ce champs quand on ne choisis pas d'option de contact

Reply:
Done!